final_project/main.ipynb

```diff
@@ -0,0 +1,8397 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from devtools import pprint\n",
+    "from sseclient import SSEClient as EventSource\n",
+    "\n",
+    "class BColor:\n",
```

Comment:
Why don't we remove unused colors?

Reply:
Adding them when needed is slower than removing))